package.json

```diff
@@ -1101,6 +1101,11 @@
           "default": true,
           "description": "Show a confirmation message before running code generated by the MongoDB participant."
         },
+        "mdb.useSampleDocsInCopilot": {
+          "type": "boolean",
+          "default": false,
+          "description": "Enable sending sample field values with the participant query command."
```

Comment:
How do you feel about `useSampleDocsInCopilotChat` to make it mention that it's for the chat?


Also same comment as above
```suggestion
        "mdb.useSampleDocsInCopilot": {
          "type": "boolean",
          "default": false,
          "description": "Enable sending sample field values with the VSCode copilot chat @MongoDB participant /query command."
```